recipes/utils/screen.yaml

```diff
@@ -1,28 +1,37 @@
-inherit: [autotools, patch]
+inherit: [autotools]
 
 metaEnvironment:
-  PKG_VERSION: 5.0.0
+    PKG_VERSION: 5.0.0
```

Comment:
Hmm...any reason for changing all the indentations? Do we have a style guide for recipes?

Reply:
I don't think there is any style guide. I just copied the formatting of the other files checked into those repositories.

Reply:
Right, there is not style guide. I guess we should add some, though. So far we mostly used 4 spaces.

But please separate style fixes from actual changes into separate commits. Conflating both makes it harder to see the actual changes.

Reply:
done